src/components/AddComponent/utils.ts

```diff
@@ -102,9 +102,9 @@ export const useComponentDetection = (
       if (loadError) {
         return loadError;
       }
-      if (loaded && !detectedComponents && cdq?.status?.conditions?.[0]?.status === 'True') {
```

Comment:
I think this might break again if in the future CDQ gets another status field, what if we instead filter the conditions by type to get the correct condition?

Reply:
Done.